test/ansible-playbooks/get_distro_version.yml

```diff
@@ -0,0 +1,14 @@
+---
+- hosts: all
+  gather_facts: yes
+  vars:
+    ansible_password: "{{ lookup('env', 'ANSIBLE_PASSWORD') | default('password') }}"
```

Comment:
Does ansible support some sort of include/shared vars or similar? It seems we need to set `ansible_{password,ssh_common_args}` in all tests so it might be useful to think about how to do it in a single place. OTOH, just two scripts right now and YAGNI (but in this case I'm pretty sure we will need it again).

Reply:
yes, is possible, but you got it - just two scripts for now … it will change